modules/orchestrator/proc-install-rhdh-with-orchestrator-helm-cli.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies
+// assembly-orchestrator-helm-modules.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-rhdh-with-orchestrator-helm-cli_{context}"]
+= Installing {product} ({product-very-short}) on {ocp-short} with the Orchestrator using the Helm chart
```

Comment:
```suggestion
= Installing {product} ({product-very-short}) on {ocp-short} with the Orchestrator using the Helm CLI
```

Otherwise, it is a bit misleading IMO, as both sections are using the Helm Chart.